charts/internal/e2e-resources/Chart.yaml

```diff
@@ -1,12 +1,12 @@
 apiVersion: v2
-version: 1.2.0-devel
+version: 1.3.0-devel
 description: This chart creates e2e resources for nri-kubernetes.
 name: e2e-resources
 
 dependencies:
   - name: kube-state-metrics
-    version: 2.13.2
-    repository: https://kubernetes.github.io/kube-state-metrics
+    version: 4.23.0
+    repository: https://prometheus-community.github.io/helm-charts
```

Comment:
Since we are making all these changes in this PR, what about updating the Helm [setup](https://github.com/newrelic/nri-kubernetes/blob/d442f74853b9f23512e7b0625dfea51955393c5e/.github/workflows/e2e.yaml#L65) repo for the e2e here? 🤔

Reply:
Updated here: [cac66c0](https://github.com/newrelic/nri-kubernetes/pull/577/commits/cac66c05853994b8e50ac0ce04dc3d1ebe757119)